tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+	"compilerOptions": {
+		"outDir": "./dist",
+		"target": "es2016", /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */
+		"module": "commonjs", /* Specify what module code is generated. */
+		"esModuleInterop": true, /* Emit additional JavaScript to ease support for importing CommonJS modules. This enables 'allowSyntheticDefaultImports' for type compatibility. */
+		"forceConsistentCasingInFileNames": true, /* Ensure that casing is correct in imports. */
+		"strict": true, /* Enable all strict type-checking options. */
+		"skipLibCheck": true /* Skip type checking all .d.ts files. */
+	}
```

Comment:
minor suggestion to boost dev speed using swc: https://typestrong.org/ts-node/docs/swc/

Reply:
Resolved in [6a21f7a](https://github.com/opengovsg/sgid-test-app/pull/94/commits/6a21f7abfb16f44d9ca5bc9fcf4c6c4e26fd6910).